doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -244,6 +244,7 @@ Common Application Framework (CAF)
     The module name can not be obtained from reference object directly, a helper function (:c:func:`module_name_get`) should be used instead.
   * Fixed the NCSDK-13058 known issue related to directed advertising in CAF.
   * Fixed NULL dereferencing in :ref:`caf_sensor_manager` during :c:struct:`wake_up_event` handling for sensors that do not support trigger.
+  * Fixed the NCSDK-13619 related to hanging buttons module in idle state if :c:struct:`wake_up_event` is consumed.
```

Comment:
```suggestion
  * Fixed the NCSDK-13619 issue where :ref: `caf_buttons` module can hang in idle state if c:struct:`wake_up_event` is called on button press.
```

Reply:
I need to add an info that the event is latar consumed. Could it be:
```
* Fixed the NCSDK-13619 issue where :ref:`caf_buttons` module can hang in idle state if :c:struct:`wake_up_event` is submitted on button press and later consumed by any other module.
```
?

Reply:
yes, this looks fine.